e2e/starter.test.js

```diff
@@ -0,0 +1,19 @@
+describe('Example', () => {
```

Comment:
What is the difference between this and app.test.ts?

Reply:
This uses detox's API and properties that are specific to detox. I believe you can configure the detox config so it runs the jest tests. I've seen examples where people update the `package.json `with ` "test": "jest"`

Ex:
```
{
  "scripts": {
     "test": "jest"
  }
}
```